ports/deathroad/Death Road to Canada.sh

```diff
@@ -35,7 +35,11 @@ else
   source "${controlfolder}/libgl_default.txt"
 fi
 
-
+# Patch final Level
+if ls $GAMEDIR/patch/canada-crossing-region.df 1> /dev/null 2>&1; then
```

Comment:
```suggestion
if [ -f "$GAMEDIR/patch/canada-crossing-region.df" ]; then
```
Doesn't rely on command output formatting and is cleaner. (LS spawns an extra process)